svthalia/concrexit
# Use queryable properties for PaymentUser

Closes #1397 

### Summary
Now, the payments app uses queryable properties for a lot of computations, like the Thalia Pay balance or whether a user has Thalia Pay enabled. This gives a massive speedup as we outsource the computation the database, that does it way more optimal than python itself

### How to test
1. Check out the payment user admin
2. Try to make thalia pay payments
